package/yast2-installation-control.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon May 15 10:49:47 UTC 2017 - igonzalezsosa@suse.com
+
+- Make the partitioning elements section reusable
```

Comment:
hm, master is stil sle12sp3 AFAIK. So a fate or bnc reference should be required ...